### PR TITLE
DOCUMENTATION: Add JSDoc info to style guide

### PR DESCRIPTION
**Pull request by tlimoncelli**

### CODE REVIEW

**Comment by tlimoncelli** [comment]
CC @j-f1

File: documentation/styleguide-code.md [hunk 1]
```diff
@@ -48,6 +48,16 @@ These are the filenames to use:
 * `providers/foo/listzones.go` -- Code for listing and creating DNS zones and domains
 * `providers/foo/dnssec.go` -- Code for DNSSEC support
 
+# Javascript annotations
```
Comment (cafferata) [inline documentation/styleguide-code.md:51]: Small typo

```suggestion
# JavaScript annotations
```

**Comment by j-f1** [comment]
Sorry, I didn’t see this PR. The only files that end up in the generated type file are the ones in [`commands/types`](https://github.com/StackExchange/dnscontrol/tree/master/commands/types). `helpers.js` doesn’t participate in any way.

**Comment by tlimoncelli** [comment]
(all the changes to the function name are in a separate PR... they were part of this PR by mistake)

**Pull request merged**